pkgs/by-name/gl/globalprotect-openconnect_2/package.nix

```diff
@@ -0,0 +1,134 @@
+{
+  perl,
+  jq,
+  fetchFromGitHub,
+  lib,
+  openconnect,
+  libsoup,
+  webkitgtk_4_0,
+  pkg-config,
+  callPackage,
+  rustPlatform,
+  glib,
+  atk,
+  gdk-pixbuf,
+  pango,
+  cairo,
+  harfbuzz,
+  gtk3,
+  zlib,
+  makeDesktopItem,
+  includeUnfreeGui ? false,
+}:
+let
+  version = "2.3.7";
```

Comment:
I have been testing the work for this package for a but and am pretty happy with it. Could we bump the version up to 2.3.9 in one go?

https://github.com/yuezk/GlobalProtect-openconnect/releases/tag/v2.3.9